src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/DownloadEvidenceNote.cshtml

```diff
@@ -0,0 +1,8 @@
+﻿@model EA.Weee.Web.ViewModels.Shared.ViewEvidenceNoteViewModel
```

Comment:
@GuidoWorkDev I'll rename the AATF view file. But with this one undo these changes as it will break the existing views. And create a new view DownloadEvidenceNotePdf.cshtml, which should be the one for the PDF. You can then rename your new action to DownloadEvidenceNotePdf